test/GCHP/integration/integrationDiffTest.sh

```diff
@@ -0,0 +1 @@
+../../shared/integrationDiffTest.sh
```

Comment:
See my earlier comment on inverting the directory structure so that we don't need to have duplicates/symlinks in GCHP and GC-Classic. It might make the relationship to shared simpler.